app/models.py

```diff
@@ -485,12 +485,13 @@ def _delete_tags_namespace(self, namespace):
         orm.attributes.flag_modified(self, "tags")
 
     def _delete_tags_alt_namespace(self, namespace):
-        for i, tag in enumerate(self.tags_alt):
-            if tag.get("namespace") == namespace:
-                with suppress(KeyError):
-                    del self.tags_alt[i]
+        if self.tags_alt:
+            for i, tag in enumerate(self.tags_alt):
+                if tag.get("namespace") == namespace:
+                    with suppress(KeyError):
+                        del self.tags_alt[i]
 
-            orm.attributes.flag_modified(self, "tags_alt")
+                orm.attributes.flag_modified(self, "tags_alt")
```

Comment:
This only needs to happen once, right? Instead of once per tag
```suggestion
            orm.attributes.flag_modified(self, "tags_alt")
```